csharp-design-patterns/AggregateDesign/AggregateDesign/DataAccess/OrderItemModel.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace AggregateDesign.DataAccess;
+
+[Table("OrderItem")]
+public class OrderItemModel
+{
+    [Key]
+    public long OrderItemId { get; set; }
+
+    public long OrderId { get; set; }
+
+    public OrderModel Order { get; set; }
+
+    public string ItemName { get; set; }
```

Comment:
This two properties can be marked as nullable reference type (using ?)

Reply:
done